.github/workflows/build-and-publish-images.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         env:
           BUILD_TYPE: ${{ inputs.BUILD_TYPE }}
   docker:
+    name: docker
```

Comment:
```suggestion
```
Unfortunately it looks like this change didn't work as intended. That's unfortunate. We should probably just revert it.

- https://github.com/rapidsai/cibuildwheel-imgs/actions/runs/5927272120/job/16070456098?pr=40

![image](https://github.com/rapidsai/cibuildwheel-imgs/assets/7400326/da78fc5e-2d8d-4328-83bd-4a9af686077e)